### PR TITLE
Adds feature flag to dump workers

### PR DESCRIPTION
**Pull request by zwolf**
Adds a feature flag check to the dump workers for classifications, workflows, subjects, and aggregations. The API will return an error that says that exports are disabled. The sidekiq jobs will hit the flag, raise an exception, and fail however many (25?) times and then be put in the dead queue where we can run them later...or not.

I have been messing with sidekiq middleware all afternoon trying to conditionally set the retry amount but it doesn't really matter. If we want to reschedule these jobs instead of basically ignoring them, I'll need to know what that schedule looks like. **If we do flip this switch, we should ensure that PFE is going to do something smart with the response.**

Closes https://github.com/zooniverse/Panoptes/issues/2252 and https://github.com/zooniverse/Panoptes/issues/2118 (more or less, depending on rescheduling thoughts).

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by zwolf** [comment]
Forgot to rescue from the raised exception and have the controller respond with something. I don't immediately see a place to test this.

File: app/controllers/api/api_controller.rb [hunk 1]
```diff
@@ -17,7 +17,8 @@ class ApiController < ApplicationController
       Subjects::Selector::MissingSubjectSet,
       Subjects::Selector::MissingSubjects,                 with: :not_found
     rescue_from ActiveRecord::RecordInvalid,               with: :invalid_record
-    rescue_from Api::LiveProjectChanges,                   with: :forbidden
+    rescue_from Api::LiveProjectChanges,
+      Api::ExportDisabled,                                 with: :forbidden
```
Comment (camallen) [inline app/controllers/api/api_controller.rb]: I don't think you need to expose this error as an api response. This error should only be detected in sidekiq after scheduling a dump to run and will just force sidekiq to retry and backoff over time, they'll just sit in the retry queue until we turn them back on.

Comment (zwolf) [inline app/controllers/api/api_controller.rb]: What happens on the front end when someone hits the button and expects to get an export? If we turn these off we should have a way of letting PFE as well as people who are hitting the API directly know that they're not gonna get anything.

Comment (camallen) [inline app/controllers/api/api_controller.rb]: it just gets queued up in the backend. It's a black box process already, the feedback then will be the same as it is already. Maybe we just put some text up there during SGL saying they may be delayed?

**Comment by zwolf** [comment]
Closes https://github.com/zooniverse/Panoptes/issues/2252

**Pull request merged**